core/captin.go

```diff
@@ -93,7 +102,7 @@ func (c Captin) Execute(e models.IncomingEvent) (bool, []captin_errors.ErrorInte
 
 	// Create dispatcher and dispatch events
 	dispatcher := outgoing.NewDispatcherWithDestinations(destinations, c.SenderMapping)
-	dispatcher.Dispatch(e, c.store, c.throttler)
+	dispatcher.Dispatch(e, c.store, c.throttler, c.documentStore)
```

Comment:
actually I am thinking if we should move the include document loading before the custom, so that we can filter destination by document attributes loaded

Reply:
Thought about it too. But it potentially increases the database load so I wasn't sure it's the best way. Totally feasible though